datastore/sql/sqliteutils.nim

```diff
@@ -250,7 +257,7 @@ proc query*(
   query: string,
   onData: DataProc): ?!bool =
 
-  let
+  var
```

Comment:
Do they need to be vars?

Reply:
Yes, unfortunately, we need to set the cached statements to `nil` on close/destroy, which requires `SQLiteDsDb` to be var.